PinpointKit/PinpointKit/Sources/Fonts.swift

```diff
@@ -32,7 +32,7 @@ public extension UIFont {
      
      - returns: A Source Sans Pro `UIFont` at the specified size and weight.
      */
-    public static func sourceSansProFontOfSize(_ fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
+    public static func sourceSansProFont(ofSize fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
```

Comment:
Yeah, I guess you’re right. There is one difference `UIFont.system` wouldn’t make a lot of intuitive sense because `system` doesn’t really sound like a specific font, whereas `UIFont.futura` definitely reads like `UIColor.red`. I think a radar for now and leaving it as is a good approach. Let me know if you want me to file it.